import "door_status.proto";

option java_package = "com.nextev.cvs_proto.protobuf.events";
option java_outer_classname = "DoorStatusChangeMessage";

message DoorStatusChangeEvent {

    optional string id = 1; // unique id, VIN for vehicle data
    optional int32 version = 2; // message protocol version
    optional uint64 sample_ts = 3; // time when door status changes
    optional DoorStatus door_status = 4; // door status
}